### PR TITLE
add Pop Falco Blog to website

### PR DESCRIPTION
**Pull request by danpopSD**
Signed-off-by: danpopsd <dan.papandrea@sysdig.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Comment by kris-nova** [comment]
so couple thoughts - wanted to get everyone's thoughts?

is the falco blog a best place for an "about me" page? i have always kept my personal work on my personal blogs or a company blog. would it make sense to consider slimming this up to falco specific content or moving the personal content to another place?

maybe we could feature contributors every month and have a "contributor of the month" section? or even feature contributors/maintainers somewhere with a small section about them? https://www.cncf.io/people/ambassadors/

just thoughts - also for legal reasons all linux foundation projects need to have a "sign off" section in the commit message (super annoying)

great read - so happy for your new title :)

https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

**Comment by danpopSD** [comment]
> so couple thoughts - wanted to get everyone's thoughts?
> 
> is the falco blog a best place for an "about me" page? i have always kept my personal work on my personal blogs or a company blog. would it make sense to consider slimming this up to falco specific content or moving the personal content to another place?
> 
> maybe we could feature contributors every month and have a "contributor of the month" section? or even feature contributors/maintainers somewhere with a small section about them? https://www.cncf.io/people/ambassadors/
> 
> just thoughts - also for legal reasons all linux foundation projects need to have a "sign off" section in the commit message (super annoying)
> 
> great read - so happy for your new title :)
> 
> https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

On Commits -  (DCO is satisfied and has a signed off by)
image

totally agree on contributors section -- see this slack (you may have missed) : https://kubernetes.slack.com/archives/CMWH3EH32/p1607013082351700

this isnt a specific "About Me"  more of an announcement of what we are doing to grow adoption, contribution, etc. and a state of the falco project and its tremendous growth which you are a great part of!

**Comment by kris-nova** [comment]
i think if this was a little more community based it would be really successful 

i've missed the last few meetings and im wondering how much of this was a decision made in the open and how much of this is just us writing down some job goals?

> more of an announcement of what we are doing to grow adoption, contribution, etc. and a state of the falco project

did the project ever decide to do any of this?

the cncf makes it easy to give project updates on their blog as well.

https://www.cncf.io/blog/2020/08/17/falco-update-whats-new-in-falco-0-25/

when i did my job update i made it a point to keep all of the work items on my personal blog - and it was later re-published on my employer blog - it seemed to go over well

**Comment by kris-nova** [comment]
i've read this a few times now - and have given this quite a bit of thought - there isn't really anything technical about the project in here to be honest - this is just a long list of ideas we seem to have to persuade people to use the software for the benefit of a company (which thank you for being honest about it)

i hate to say it - this is a no from me until we can get something more concrete about the project in here

- what features are important? 
- why are they important?
- how will we build them?
- why will we build them?
- do you have consensus for these features?
- has anyone else looked at them?
- what about testing them? 

it's a great community blog that shares something deeply personal to you - but i just am not sold that a technical blog is the right place for this 

---

keeping this open to give others a chance to respond

File: content/en/blog/pop-joins-falco.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+ - Work within the cloud-native community to see where Falco is being used and shepherd their experience so that they also are also advocates for the power of Falco!
+
+ - Assist current adopters with Falco and ensure they have a sounding board for their integration and use of Falco so that their adoption grows even further!
+
+ #### Graduation - Help the project grow from Incubated to Graduated CNCF project
+
+ - Work under the guidance of the CNCF in order to ensure Falco is compliant with the requirements to graduate.
+
+ - Ensure target requirements are being met in order to enrich the user experience in a way that makes the project’s graduation a no-brainer
```
Comment (kris-nova) [inline content/en/blog/pop-joins-falco.md:54]: can you enumerate these targets?

File: content/en/blog/pop-joins-falco.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ #### Adoption -  Work with current adopters and find more end users using Falco
+
+
+ - Work within the cloud-native community to see where Falco is being used and shepherd their experience so that they also are also advocates for the power of Falco!
+
+ - Assist current adopters with Falco and ensure they have a sounding board for their integration and use of Falco so that their adoption grows even further!
+
+ #### Graduation - Help the project grow from Incubated to Graduated CNCF project
+
+ - Work under the guidance of the CNCF in order to ensure Falco is compliant with the requirements to graduate.
```
Comment (kris-nova) [inline content/en/blog/pop-joins-falco.md:52]: what makes falco a graduation ready project?
it's more than just checking boxes prescribed by a dedicated marketing committee under the linux foundation.

File: content/en/blog/pop-joins-falco.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+ - Ensure target requirements are being met in order to enrich the user experience in a way that makes the project’s graduation a no-brainer
+
+ My goal as the Lead of Open Source Community and Ecosystem is to spread even more awareness around the Falco project and the magnificent work the team is doing!  We want YOU the community to join the Falco project, tell us how you are using Falco, contribute in places like the Falco core engine, rulesets, integrations, etc.  We would like to make it easy for you to contribute. We want to celebrate your contributions!  We want you to use and triumph in using the Falco engine, rules, and outputs. Use the subprojects like sidekick, event generator and docs page, let us know what would make your experience even better.  If you are a company or technology partner that is using Falco… LET US KNOW, let’s help you adopt the power of Falco to meet your end goals!
+
+ Reach out to me on twitter @danpopnyc or on the [Falco slack](https://kubernetes.slack.com/archives/CMWH3EH32)
```
Comment (kris-nova) [inline content/en/blog/pop-joins-falco.md:58]: a lot of contributors have put a lot of hours into https://github.com/falcosecurity/community 

maybe we could add more contact detail there?

File: content/en/blog/pop-joins-falco.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+ I chose to move from my Field CTO position to Falco because I believe in the technology and I love the cloud-native community.  I started a [podcast](https://github.com/danpopSD/popcast#show-follow-and-subscribe-to-our-youtube-channel--twitter) the popcast with danpop at the beginning of this pandemic as a love letter to the community and to connect at a human level with the leaders who created its awesome tech, beyond just their code.  I am a member of the Kubernetes Contributor group, who eats, sleeps and breathes how to best contribute, adopt, and support what I believe is the greatest technical community in the world -- Kubernetes -- and its constellation of supporting projects.  Being able to work with the amazing ecosystem within the CNCF and the various surrounding projects is so inspiring and I cannot wait to help nurture adoption, contribution, and end-user success.  This is what gets me pumped.  I am such a big fan of these projects and there is so much power and possibility if we work together in the true spirit of collaboration and COMMUNITY.
+
+ ***Falco’s Progress***
```
Comment (kris-nova) [inline content/en/blog/pop-joins-falco.md:22]: this whole section is just vanity metrics

what has falco done? what features have been impactful? 

you mention a lot of contributors/maintainers but what have they done?

- Jonah enabled our builds and hosting our drivers
- Spencer has been the biggest voice for the helm chart
- Leo and Lore have built a state of the art build system from the ground up
- Leo has basically taken over the helm chart and has single handily debugged the drops using his event generator framework

**Comment by kris-nova** [comment]
wonder if @caniszczyk @idvoretskyi or Nancy would be interested in a big exciting press event like this for cncf.io?

**Comment by danpopSD** [comment]
i have submitted this blog as well to CNCF blogs.  thank you for these opinions and feedback.  appreciate the literal dissection being done of a piece i was excited to share with the community in terms of growing our contribution and adoption.   considering this PR closed.

**Comment by danpopSD** [comment]
image
 Thank you CNCF for publishing this!

**Comment by idvoretskyi** [comment]
So this is solved on the CNCF side, awesome :)

**Comment by danpopSD** [comment]
thank you @idvoretskyi yes.  (hoping it can be pushed out sooner than December 21st though) ;)

this was meant to be a document to promote the great progress that was made and the project itself and get people interested in contribution regardless of where it ended up.

**Pull request closed without merging**